collectd/packages.sls

```diff
@@ -0,0 +1,13 @@
+{% from "collectd/map.jinja" import collectd_settings with context %}
+
+include:
+  - collectd
```

Comment:
I've just remembered. It's required because in the following situation.
* You enabled some collectd plugin, which requires additional package
* You see it doesn't work.
* You install the package.
* Here you need to restart collectd in order to it loads newly installed libraries.
Without this line collectd won't be restarted